res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+            <title>
+                <de>Superweib und Superkerl versöhnen sich wieder</de>
+                <en>Superwoman and Superguy reconcile again</en>
+            </title>
+            <description>
+                <de>Superkerl: "Wir sind wieder glücklich. Unsere Versöhnung war super.</de>
+                <en>Superguy: "We are happy again. Our reconciliation was super."</en>
+            </description>
+            <effects>
+                <!-- "morgen oder uebermorgen, 18-22 Uhr" -->
+                <effect trigger="happen" type="triggernewschoice" choose="or" probability="50" time="2,1,2,18,22"
+                    news1="cujo-news-superkerl-08a" probability1="50"
+                    news2="cujo-news-superkerl-08b" probability2="50"
+                />
+            </effects>
+            <data genre="1" price="1.0" quality="40" />
+        </news>
+        <news id="cujo-news-superkerl-08a" type="2" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+            <title>
+                <de>Superkerl und Superweib lassen sich scheiden</de>
+                <en>Superguy and Superwoman get a divorce</en>
+            </title>
+            <description>
+                <de>Superboy leidet unter Trennung. Er will einen Supervater und eine Supermutter. Das unappetitliche Essen stört ihn nicht.</de>
+                <en>Superboy suffers from separation. He wants a super father and a super mother. The unappetizing food does not bother him.</en>
+            </description>
+            <effects />
+            <data genre="1" price="1.0" quality="70" />
+        </news>
+        <news id="cujo-news-superkerl-08b" type="2" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+            <title>
+                <de>Superkerl und Superweib lassen sich scheiden</de>
+                <en>Superguy and Superwoman get a divorce</en>
+            </title>
+            <description>
+                <de>Superboy leidet unter Trennung. Er will einen Supervater und eine Supermutter. "Ich brauche kein neues Regal".</de>
+                <en>Superboy suffers from separation. He wants a super father and a super mother. "I do not need a new shelf".</en>
+            </description>
+            <effects />
+            <data genre="1" price="1.0" quality="70" />
+        </news>		
+        <!-- "Fifa-WM 2022 in Katar" -->
+        <news id="cujo-news-quantar-01" type="0" thread_id="cujo-news-quantar" creator="8827" created_by="Cujo">
+            <title>
+                <de>Fufa vergibt heute die Fußball-WM 2022</de>
+                <en>Fufa awards the 2022 World Cup today</en>
+            </title>
+            <description>
+                <de>Heute gibt die Fufa den Austragungsort der Fußball-WM 2022 bekannt. SUA, Pajan und Intralien gelten als Favoriten. Der Wüstenstaat Quantar gilt als krasser Außenseiter.</de>

```

Comment:
Vergabe demnächst? Dann kann die Nachfolgenachricht etwas später kommen.

Reply:
Was wäre der Vorteil? Der Gedanke war, dass Nachrichtenagenturen oftmals morgens auf geplante Ereignisse hinweisen, die im Tagesverlauf bevorstehen.

Reply:
Wir nutzen ja echte Laendernamen ("Deutschland, USA, GB ..."), ich denke die koennen wir hier auch nutzen.

Reply:
diese "Vornachricht" finde ich auch OK - vielleicht mit etwas "geringeren Attributen". Und dann halt als Folgenachricht die "Sensation" (oder eher nicht) am gleichen Tag.

Ist ein wenig wie mit einem Wahlsonntag, da wird ja auch nicht nur ueber das Ergebnis berichtet :)

Reply:
Es ging eher um die Formulierung. Vorankündigung mit "heute" nur, wenn die Hauptnachricht dann auch wirklich am selben Tag kommt (Trigger muss ich nochmal prüfen).
Mit "Mit der Ankündigung der Vergabe wird in Kürze gerechnet" kann man sich den Nachfolgezeitpunkt etwas offener lassen.

(Und ein Kommentar heißt auch nicht immer, dass was geändert werden muss.)